### ISSUE TITLE
Upload and Publish campaigns

### ISSUE THREAD

**Issue by dev-rajneeshkumar** [hlxsites/moleculardevices#1114]
Upload and Publish campaigns folder to make urls like below work

https://www.moleculardevices.com/campaigns/en/chart-your-own-course/
https://www.moleculardevices.com/campaigns/fr/tracez-votre-propre-chemin/

Regards,
Rajneesh

**Comment by dev-rajneeshkumar** [comment]
I'll upload the folder to Franklin root then we need to make bulk publish using @andreituicu's script like we did for KB images/pdfs. Also this folder contains some image names with caps or _ so need to set redirects for them as well.

**Comment by alexcarol** [comment]
@dev-rajneeshkumar is this still pending? Do you need anything from our side to merge #1115?

**Comment by mhaack** [comment]
@alexcarol yes it is. The PR must be updated. I only English version, other languages must be adjusted.
